### PR TITLE
chore: local_wandb_server.py --interactive

### PR DESCRIPTION
**Pull request by timoffex**
```
❯ python tools/local_wandb_server.py start manual -i
Registry [us-central1-docker.pkg.dev]: 
Repository [wandb-production/images/local-testcontainer]: 
Tag [master]: 
--pull [always]: 
local_wandb_server.py: Running command: docker run --detach --pull always -e WANDB_ENABLE_TEST_CONTAINER=true --name wandb-local-testcontainer --volume wandb-local-testcontainer-vol:/vol --publish 8080 --publish 9015 --platform linux/amd64 --rm us-central1-docker.pkg.dev/wandb-production/images/local-testcontainer:master
master: Pulling from wandb-production/images/local-testcontainer
2b55860d4c66: Already exists 
8e058277d049: Already exists 
...
```

### CODE REVIEW

File: tools/local_wandb_server.py [hunk 1]
```diff
@@ -356,10 +381,22 @@ def _start_container(
...
+    registry = "us-central1-docker.pkg.dev"
+    repository = "wandb-production/images/local-testcontainer"
+    tag = "master"
+    pull = "always"  # Always get latest image.
+
+    if interactive:
+        registry = click.prompt("Registry", default=registry)
+        repository = click.prompt("Repository", default=repository)
+        tag = click.prompt("Tag", default=tag)
+        pull = click.prompt("--pull", default=pull)
```
Comment (kptkin) [inline tools/local_wandb_server.py:395]: how about using click.Choice here as pull can only be one of predefined options

Comment (timoffex) [reply tools/local_wandb_server.py]: done!

**Review by kptkin — APPROVED** [review]
Nice! Mind updating contrib doc?

**Comment by timoffex** [comment]
> Nice! Mind updating contrib doc?

We don't seem to have a section on system tests or about the previous usage. Plus external contributors won't be able to run the local-testcontainer anyway. Is this the right place to put that?

**Comment by kptkin** [comment]
> > Nice! Mind updating contrib doc?
> 
> We don't seem to have a section on system tests or about the previous usage. Plus external contributors won't be able to run the local-testcontainer anyway. Is this the right place to put that?

fair, so maybe update the unmock doc or just add a new internal doc so we can reference devs?

**Comment by timoffex** [comment]
> > > Nice! Mind updating contrib doc?
> > 
> > 
> > We don't seem to have a section on system tests or about the previous usage. Plus external contributors won't be able to run the local-testcontainer anyway. Is this the right place to put that?
> 
> fair, so maybe update the unmock doc or just add a new internal doc so we can reference devs?

will do!

**Pull request merged**